### PR TITLE
WICKET-6438 - 8.x reference guide needs a few minor improvements

### PR DESCRIPTION
**Pull request by ozeray**
Minor fix in helloWorld.adoc: Removed irrelevant line to avoid warning during maven packaging phase:
WARNING: image to embed not found or not readable: F:/wicket/wicket-user-guide/src/main/asciidoc/images/uml-component.svg

Minor fix in contributing.adoc: Revised the generated document target directory for the user guide documentation.

Cropped top and bottom blank parts in comsysto-logo.png to have the Introduction page fit in one page in PDF. Made minor changes in helloWorld_2.adoc, helloWorld_3.adoc and introduction.adoc.

### CODE REVIEW

**Comment by ozeray** [comment]
Added github forking step into Contribution section.

Made a few minor changes to adapt with this new step.

**Comment by ozeray** [comment]
Closed by fault. Sorry. Reopened the pull request.

File: wicket-user-guide/src/main/asciidoc/introduction.adoc [hunk 1]
```diff
@@ -27,8 +26,7 @@ Editors:
 *Joachim Rohde*
 
 
-*PS*: this guide is based on Wicket 6. However if you are using an older version you should find this guide useful as well, but it's likely that the code and the snippets won't work with your version.
-
+*PS*: this guide is based on Wicket 6. However if you are using an older version you should find this guide useful as well, but it's likely that the code and the snippets won't work with your version. +
```
Comment (martin-g) [inline wicket-user-guide/src/main/asciidoc/introduction.adoc]: Note to the one merging this PR: It should be `Wicket 8` instead of `6`

**Comment by ozeray** [comment]
Implemented Martin's diff request. Made some more improvements on 8.x reference documentation.

**Comment by bitstorm** [comment]
hi @ozeray ,

please close this issue as i don't have the rights to do it

**Comment by klopfdreh** [comment]
The changes of the PR are integrated with commit 075d8371ef3eb2e5f3bdadf008ae904cafd222b3

**Comment by bitstorm** [comment]
@ozeray ,

could you please close this PR?

Thank you!

**Pull request closed without merging**